openfl-workspace/default/plan/plan.yaml

```
@@ -9,6 +9,7 @@ aggregator :
     best_state_path : save/best.pbuf
     last_state_path : save/last.pbuf
     rounds_to_train : 10
+    tensor_board    : false
```

```suggestion
    tensorboard    : false
```
We probably should keep naming from [tensorflow/tensorboard](https://github.com/tensorflow/tensorboard) and [lanpa/tensorboardX](https://github.com/lanpa/tensorboardx).

---

Agree with @itrushkin for naming